windows/benji.py

```diff
@@ -99,13 +99,14 @@ def events(frame,put):
 				print("By ", status.user.screen_name, " at ", status.user.created_at)
 				
 	#Get friends from twitter
-	elif link[-3] == "follow" and link[-1] == "twitter":
-		auth = OAuthHandler(twitterCredentials.consumer_key, twitterCredentials.consumer_secret)
-		auth.set_access_token(twitterCredentials.access_token, twitterCredentials.access_secret)
-		api = tweepy.API(auth)
-		for friend in tweepy.Cursor(api.friends).items():
-			print("\nName: ", json.dumps(friend.name), " Username: ", json.dumps(friend.screen_name))
-	
+	elif link[-1] == "twitter":

```

Comment:
I still don't understand what was wrong with the previous if condition, you are anyway checking the `link[-1]==twitter` again in the nested if condition.

Reply:
previous if condition was giving me Error `List Index Out of Range :` as per below:
![screenshot 68](https://user-images.githubusercontent.com/25661686/43022899-78a7a362-8c86-11e8-8a10-e1b2d4ede363.png)

Instead of generating issue , i Solved it.